### PR TITLE
Add toyplot

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
Needed for https://github.com/bioconda/bioconda-recipes/pull/21584

File: recipes/toyplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  imports:
+    - toyplot
+    - toyplot.reportlab
+
+about:
+  home: "http://toyplot.readthedocs.org"
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "A modern plotting toolkit supporting electronic publishing and reproducibility."
```
Comment (synapticarbors) [inline recipes/toyplot/meta.yaml:48]: add `dev_url: https://github.com/sandialabs/toyplot`

File: recipes/toyplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f3c07ed64609fa8747210421717b241c808da2fc5d296615c83bf422faca9e90
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/toyplot/meta.yaml:18]: It looks like `host` likely only needs `python` and `pip`

File: recipes/toyplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - six
+
+test:
+  imports:
+    - toyplot
+    - toyplot.reportlab
+
+about:
+  home: "http://toyplot.readthedocs.org"
+  license: BSD
```
Comment (synapticarbors) [inline recipes/toyplot/meta.yaml:45]: Use a spdx compatible identifier:

https://spdx.org/licenses/

File: recipes/toyplot/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  license_family: BSD
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/toyplot/meta.yaml:47]: Add a comment here with the following link as a reminder to include the upstream license if it is added in a future release:

https://github.com/sandialabs/toyplot/issues/197

**Comment by dpryan79** [comment]
@synapticarbors I've tended to the comments.

**Comment by synapticarbors** [comment]
So a few things:

- It looks like there was a `custom-inherit` feedstock already (https://github.com/conda-forge/custom-inherit-feedstock)
- `custom_inherit` is suffering from an error that is keeping new feedstocks from building.

**Comment by dpryan79** [comment]
Since the feedstock hasn't been created yet, I'll delete custom_inherit. @synapticarbors You don't need to wait for the CI to pass, it already did before deleting the errant custom_inherit.

**Pull request merged**